docs/release_process.md

```diff
@@ -40,8 +40,18 @@ Let's make sure there are no migration conflicts with the previous release:
     $ git checkout master
     $ make migrate
```

Comment:
Hum, I had the documentation on readthedocs.org in mind. But I've just check and the default one is the `lastest` version not the `stable` one.

So yes, let's keep it for now and remove it after the `0.22.0` release.